src/components/CookiesAlert/CookiesAlert.js

```diff
@@ -20,9 +18,9 @@ export const CookiesAlert = ({
   content, confirmCookies, isVisible,
 }) => (
   <Wrapper isVisible={isVisible}>
-    <SectionHeading>
+    <CookieHeading>
```

Comment:
Why not just use the `as="p"` prop instead of duplicating the entire component under another name?

Reply:
Yes, you're right we could use the as="p" prop, but we have talked to the client, and he is fine with it.

Reply:
Sure, when the code is on the client's repo, they'll have a say in regards to code quality. Until then - [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

Reply:
@rsmalarz please adhere to kjhank comment and correct pull request in order to commit. Thank you.

Reply:
@kjhank can you change the pull request as required to comply with DRY code quality?